backend/clickhouse/console_messages_parser.go

```diff
@@ -0,0 +1,73 @@
+package clickhouse
+
+import (
+	"encoding/json"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+	e "github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+)
+
+type Message struct {
+	Type  string `json:"type"`
+	Trace []struct {
+		ColumnNumber int    `json:"columnNumber"`
+		LineNumber   int    `json:"lineNumber"`
+		FileName     string `json:"fileName"`
+		FunctionName string `json:"functionName,omitempty"`
+		Source       string `json:"source"`
+	} `json:"trace"`
+	Value []string `json:"value"`
+	Time  int64    `json:"time"`
+}
+
+type Messages struct {
+	Messages []Message `json:"messages"`
+}
+
+func ParseConsoleMessages(projectID int, sessionSecureID string, messages string) ([]*LogRow, error) {
+	messagesParsed := Messages{}
+	if err := json.Unmarshal([]byte(messages), &messagesParsed); err != nil {
+		return nil, e.Wrap(err, "error decoding message data")
+	}
+
+	var logRows []*LogRow
+	for _, message := range messagesParsed.Messages {
+		logRow, err := makeLogRow(projectID, sessionSecureID, message)
+		if err != nil {
+			// If there's an issue with parsing, we'll log for investigation and try the next one
+			log.WithError(err).Error("failed to parse log message")
+			continue
+		}
+		logRows = append(logRows, logRow)
+	}
+	return logRows, nil
+}
+
+func makeLogRow(projectID int, sessionSecureID string, message Message) (*LogRow, error) {
+
+	// Try unquoting data from console.log
```

Comment:
The previous [code](https://github.com/highlight/highlight/blob/59afaa33977fc81bb42c5867a81fd28e445cbfee/backend/clickhouse/logs.go#L141-L147) was trying to be smart and pluck out the timestamp.

This isn't very robust since there's no real guarantee on the order or length of arguments passed to `console.log`.

Now we just join the messages together and stuff it into `Body`.

The final output looks like this:
![Screenshot 2023-02-01 at 2 14 10 PM](https://user-images.githubusercontent.com/58678/216165537-9dc25a33-dd42-48aa-9f39-4337c9cfd0ea.png)

That `[timestamp]` prefix is coming from here:
https://github.com/highlight/highlight/blame/8311ff5036d19d6de74d80a546281f5fb92cdec9/sdk/client/src/logger.ts#L12-L16

I'm not really sure why we even do this (since something like `1675285833795` isn't really readable by a human) but it only happens when running the client with the `debug` option.